policyengine_us/variables/gov/states/md/tax/income/deductions/md_itemized_deductions.py

```diff
@@ -7,10 +7,36 @@ class md_itemized_deductions(Variable):
     label = "Maryland itemized deductions"
     unit = USD
     definition_period = YEAR
-    reference = (
-        "https://govt.westlaw.com/mdc/Document/N05479690A64A11DBB5DDAC3692B918BC?viewType=FullText&originationContext=documenttoc&transitionType=CategoryPageItem&contextData=(sc.Default)"
-        "https://www.marylandtaxes.gov/forms/21_forms/Resident_Booklet.pdf#page=5"
-        "https://www.marylandtaxes.gov/forms/22_forms/Resident_Booklet.pdf#page=5"
-    )
+    reference = [
+        "https://govt.westlaw.com/mdc/Document/N05479690A64A11DBB5DDAC3692B918BC?viewType=FullText&originationContext=documenttoc&transitionType=CategoryPageItem&contextData=(sc.Default)",
+        "https://www.marylandtaxes.gov/forms/21_forms/Resident_Booklet.pdf#page=5",
+        "https://www.marylandtaxes.gov/forms/22_forms/Resident_Booklet.pdf#page=5",
+        "https://mgaleg.maryland.gov/2025RS/Chapters_noln/CH_604_hb0352e.pdf#page=160",  # Maryland House Bill 352 - Budget Reconciliation and Financing Act of 2025
+    ]
     defined_for = StateCode.MD
-    adds = ["itemized_deductions_less_salt", "capped_property_taxes"]
+
+    def formula(tax_unit, period, parameters):
+        # Base itemized deductions (federal itemized deductions less SALT plus capped property taxes)
+        base_itemized = add(
+            tax_unit,
+            period,
+            ["itemized_deductions_less_salt", "capped_property_taxes"],
+        )
+
+        # Apply income-based phase-out if applicable
+        p = parameters(period).gov.states.md.tax.income.deductions.itemized
+        if p.phase_out.applies:
+            filing_status = tax_unit("filing_status", period)
+            md_agi = tax_unit("md_agi", period)
```

Comment:
This also looks like federal AGI to me in the [bill text](https://mgaleg.maryland.gov/2025RS/Chapters_noln/CH_604_hb0352e.pdf#page=167):